src/main/java/service/operation/ReturnOperation.java

```diff
@@ -0,0 +1,22 @@
+package service.operation;
+
+import data.db.Storage;
+import java.util.Map;
+import model.FruitTransaction;
+
+public class ReturnOperation implements OperationHandler {
+
+    @Override
+    public void operation(FruitTransaction.Operation operation, FruitTransaction fruitTransaction) {
+        if (fruitTransaction.getQuantity() <= 0) {
+            return;
```

Comment:
There is no need to check if the quantity is less than or equal to zero since the return operation should not occur if there is nothing to return. This validation should be performed before calling the operation, or you should throw an exception to indicate that the operation is invalid.